fullscreen.bs

```diff
@@ -11,6 +11,11 @@ Markup Shorthands: css no
 </pre>
 
 <pre class=link-defaults>
+spec:html
+    type:dfn; text:allowed to use
```

Comment:
FYI Reviewers: This is to fix the build as it's currently broken (see #246 for a standalone PR to fix that if that's preferable)